examples/advanced/sklearn-svm/README.md

```diff
@@ -0,0 +1,116 @@
+# Federated SVM with Scikit-learn
+
+## Introduction to Scikit-learn, tabular data, and federated SVM
+### Scikit-learn
+This example shows how to use [NVIDIA FLARE](https://nvflare.readthedocs.io/en/main/index.html) on tubular data.
+It uses [Scikit-learn](https://scikit-learn.org/),
+which is a widely used open source machine learning library that supports supervised and unsupervised learning.
+### Tabular data
+The data used in this example is tabular in a format that can be handled by [pandas](https://pandas.pydata.org/), such that:
+- rows correspond to data samples
+- first column represents the label 
+- the other columns cover the features.    
+
+Each client is expected to have 1 local data file containing both training and validation samples. To load the data for each client, the following parameters are expected by local learner:
+- data_file_path: string, full path to the client's data file 
+- train_start: int, start row index for training set
+- train_end: int, end row index for training set
+- valid_start: int, start row index for validation set
+- valid_end: int, end row index for validation set
+
+### Federated SVM
+The machine learning algorithm shown in this example is [SVM with RBF kernel](https://scikit-learn.org/stable/modules/generated/sklearn.svm.SVC.html).
```

Comment:
Can we make it general here?
The algorithm shown is [Support Vector Classification].

Because I notice we can pass in diff. kernel functions to it, so maybe later in the config section you add a line saying we are chosing rbf kernel here.

Reply:
good point, changed